components/api_version/src/lib.rs

```diff
@@ -299,19 +309,19 @@ mod tests {
             KeyMode::TiDB
         );
         assert_eq!(
-            ApiV2::parse_range_mode((Some(b"x\0a"), Some(b"x\0z"))),
+            ApiV2::parse_range_mode((Some(b"x\x00\x00\x00a"), Some(b"x\x00\x00\x00z"))),
```

Comment:
Suggest to use variables instead of direct use ```\x00\x00\x00``` and ```\xff\xff\xff```

Reply:
This requires us to add many syntax noises to build literal test cases. The existed prefix `x` or `r` are hard-coded in every case, I think keeping the test case KISS should be enough.